src/components/data/listing/Listing.js

```diff
@@ -452,7 +472,11 @@ function Listing(props) {
         }
     };
 
-    const isLoading = isQueryLoading([isFetching, removeResourceStatus]);
+    const isLoading = isQueryLoading([
+        isFetching,
+        removeResourceStatus,
+        isFetchingDefaultsMapping,
+    ]);
```

Comment:
Updated to account for the new API call to fetch the defaults mapping. Still not 100% that I like having the additional API call here, but I'm unsure about where else to put it so that it will get reloaded periodically.